ibis/backends/tests/test_array.py

```diff
@@ -1061,9 +1056,9 @@ def test_array_flatten(backend, flatten_data, column, expected):
     data = flatten_data[column]
     t = ibis.memtable({column: data["data"]}, schema={column: data["type"]})
     expr = t[column].flatten()
-    result = backend.connection.execute(expr)
+    result = backend.connection.to_pyarrow(expr)
```

Comment:
Why this change here? Does `con.to_pandas` not work, but `con.to_pyarrow().to_pandas()` does work? If so, we should fix that.

Reply:
It works, but `to_pyarrow` reproduces the issue due to polars having different typing behavior for `concat_list` versus `flatten`.